### PR TITLE
Non-DL results of DL sprint-week project

### PR DESCRIPTION
**Pull request by James-Large**
### Reference Issues/PRs

See closed dl4tsc PR #88 

Separate PR for non-deeplearning based items from sprint week: 

* [Column ensembler](https://github.com/alan-turing-institute/sktime/compare/dev...James-Large:column-ensembler?expand=1#diff-26b37cfc459dc1d59e89e77cb978bb50) and [tests](https://github.com/alan-turing-institute/sktime/compare/dev...James-Large:column-ensembler?expand=1#diff-2cd43f45cc901b624d9f1829a58e4e65)
* Japanese Vowels [baked in dataset](https://github.com/alan-turing-institute/sktime/compare/dev...James-Large:column-ensembler?expand=1#diff-bc80aab8987a0c8731230debc924bea2)

### CODE REVIEW

**Comment by James-Large** [comment]
Hmm, was a quick port over and edit of the files and pycharm didn't complain. Will have a deeper look, presumably some imports have moved around

**Comment by James-Large** [comment]
Resolved a boss import error, this is now under sktime.dictionary_based.boss, was it meant for sktime.classifier.dictionary_based.boss ? 

Still errors though...

**Comment by James-Large** [comment]
Yay, it was that the JapaneseVowels ts file weren't initially created correctly with full meta-data

**Comment by TonyBagnall** [comment]
I may have put boss in the wrong place, I'll check

File: sktime/contrib/column_ensembler.py [hunk 1]
```diff
@@ -0,0 +1,459 @@
...
+from sklearn.preprocessing.label import LabelEncoder
+
+from sktime.classifiers.base import BaseClassifier
+
+
+
+
+
+#TODO: should probably inherit from a meta estimator in SKlearn
+class ColumnEnsembler(BaseClassifier):
```
Comment (mloning) [inline sktime/contrib/column_ensembler.py]: would rename to `ColumnEnsembleClassifier`

File: sktime/contrib/column_ensembler.py [hunk 2]
```diff
@@ -0,0 +1,459 @@
...
+    """
+    if hasattr(column, 'dtype') and np.issubdtype(column.dtype, np.bool_):
+        return not column.any()
+    elif hasattr(column, '__len__'):
+        return len(column) == 0
+    else:
+        return False
+
+
+class SimpleColumnEnsembler(ColumnEnsembler):
```
Comment (mloning) [inline sktime/contrib/column_ensembler.py]: perhaps rename to `HomogeneousColumnEnsembleClassifier`?

File: sktime/classifiers/tests/test_columnensembler.py [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+
+def test_simple_column_ensembler():
+    X_train ,y_train  = _load_dataset("JapaneseVowels", "TRAIN", True)
+    X_test, y_test = _load_dataset("JapaneseVowels", "TEST", True)
+
+    cts = SimpleColumnEnsembler(KNNTSC(n_neighbors=1))
+
+    cts.fit(X_train, y_train)
+    print(cts.score(X_train, y_train))
```
Comment (mloning) [inline sktime/classifiers/tests/test_columnensembler.py]: would remove print statement here and throughout the test file and replace it with some sensible assertion, e.g. checking whether shape of y_pred matches shape of y_test or comparing ColumnEnsembler results with manual ensembling

**Pull request merged**